### ISSUE TITLE
Clarification for the dataset statistics

### ISSUE THREAD

**Issue by BierOne** [zjunlp/EasyEdit#229]
Hi, there!

Nice work! I noticed that the dataset statistics presented in your two survey papers (2024'Comprehensive Study' and 2023'Problems, Methods, Opportunities') are inconsistent. 

For instance, in KnowEdit, the ZsRE dataset is listed with only **1230** test samples. However, in the 2023 version, the same dataset comprises **293+385+1,037** test samples (Table 8 in 2023 paper).

Similar inconsistencies also appear to exist in the Counterfact dataset. Could you please clarify the differences between these versions? The presence of multiple datasets and versions can be confusing, especially for newcomers in the field.

Also, I am a little curious about the splits of datasets. In KnowEdit, are the ZsRE test samples sourced from its original validation split? It seems that there isn't a designated validation split in the KnowEdit (Table 4 in 2024 'Comprehensive Study'). Does this imply that a validation set isn't necessary in the KnowEditing experiments?

Thank you in advance for addressing these questions. Look forward to hearing from you.

Best,
Yibing

**Comment by littlefive5** [comment]
Hello, thanks for your attention to our work.
1. In the 2023' paper, our main goal is to analyze the performance of current editing methods and explore the limitations, so we use the original ZsRE and Counterfact settings and provide some extensions like portability. Further in 2024', we combine our EMNLP paper and 'Evaluating the Ripple Effects of Knowledge Editing in Language Models (TACL 2024)' together and build a new benchmark to evaluate the editing method comprehensively. Hence, here we filtered some data from the 2023 EMNLP's data and created more settings (discussed in the paper).  Also, the counterfact datasets are different in the two papers, in KnowEdit, we used the version in 'Ripple Effects' (actually, we named it $Wiki_{counterfact}$), and in EMNLP, we used the version in ROME. The reason we selected $Wiki_{counterfact}$ is that it contains more evaluation aspects and the ROME's version bears some conflicts and noises in some data.

2. Actually, under the knowledge editing set, validation split is not necessary, even for the train set. Since our goal is to edit one case, there's no need to train for many methods except for meta-learning methods.

**Comment by littlefive5** [comment]
Thanks for your advise. We would provide a validation set in the future, but currently, only the convsent dataset has the validation data, and currently we only make sure that the test-split contains the  both the Portability and Locality evaluations.

**Comment by zxlzr** [comment]
Dear Yibing,

Thanks for your constructive suggestions. We have updated the REAME to help newcomers better understand KnowEdit. We will continue maintain the EasyEdit and KnowEdit. Feel free to provide more feedback.

Best,

Team of EasyEdit

**Comment by BierOne** [comment]
Dear all,

Many thanks for your efforts. Look forward to seeing the updated version.
